production/daemonset-admin.yml

```diff
@@ -0,0 +1,31 @@
+#
```

Comment:
Is it related to the production template? Then `jaeger-production-daemonset-admin.yml` If not without production?